docs/index.md

```diff
@@ -114,7 +114,6 @@ There are a number of ways available to specify how often the task is called.
 | ->yearly('12:34am')           | Runs the first day of the year.                                       |
 | ->weekdays('1:23pm')          | Runs M-F at 12:00 am unless time passed in.                           |
 | ->weekends('2:34am')          | Runs Saturday and Sunday at 12:00 am unless time passed in.           |
-| ->timezone('America/Chicago') | Specifies a timezone to use when determine if it runs.                |
```

Comment:
Somehow this messed up the table formatting.
<img width="1041" alt="Screen Shot 2020-12-09 at 10 07 38 AM" src="https://user-images.githubusercontent.com/17572847/101647208-7447a800-3a06-11eb-9f71-181ab95353f3.png">